library/ssl_tls.c

```diff
@@ -5956,6 +5970,17 @@ static unsigned char ssl_serialized_session_header[] = {
  * The order is the same as in the definition of the structure, except
  * verify_result is put before peer_cert so that all mandatory fields come
  * together in one block.
+ *
+ * In TLS/DTLS 1.3 the ticket contains the following content:
+ *
+ *  - ticket creation time ( start )
```

Comment:
This lists only one "ticket creation" value, but below we're writing both `session->start` and `session->ticket_received`. What is the difference, if any? Do we need to write both? If so, the documentation here should reflect that.

Reply:
Fixed it. Added both